src/reducers/AddSiteReducer.js

```diff
@@ -116,11 +117,13 @@ export default (state = INITIAL_STATE, action) => {
             return {...state, siteReadyForUpload};
 
         case ADD_SITE_SUCCESS:
-            return INITIAL_STATE;
+            return {...INITIAL_STATE, sitesShouldUpdate: true};
```

Comment:
This is kind of a hack for now... whenever we add a new site, fire off a `sitesShouldUpdate` boolean which will tell the `SearchScreen` to fetch the latest sites. We can revisit this if there's a cleaner way later.